### ISSUE TITLE
Event ping on page 15

### ISSUE THREAD

**Issue by claudia1204** [kinabalu/websocketsbook#4]
Hello~ 
First thanks for your knowledge sharing in your book, it helps me a lot to understand websocket~~
But I found a little mistake here on page 15, it says "**PING frames are sent out by the server only, and browser implementations should send back PONG frames in response**."
in [mdn](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API/Writing_WebSocket_servers#Pings_and_Pongs_The_Heartbeat_of_WebSockets), it  says that "**At any point after the handshake, either the client or the server can choose to send a ping to the other party.**"
Wish your book is getting better and better

**Comment by kinabalu** [comment]
@ClaudiaHunter you're right, that section was meant to reference the lack of a JavaScript API for sending PING frames from the browser, but the browser itself of course will send PINGs from the specific implementation.

I've got a note to talk with O'Reilly about adding that to the Errata but their system is currently down.  Thank you for the feedback!

**Comment by claudia1204** [comment]
@kinabalu You're welcome~ O(∩_∩)O
